esphome/components/tcl112/climate.py

```diff
@@ -0,0 +1,38 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import climate, remote_transmitter, sensor
+from esphome.const import CONF_ID, CONF_NAME, CONF_SENSOR
+
+AUTO_LOAD = ['sensor']
+
+tcl_ns = cg.esphome_ns.namespace('tcl112')
```

Comment:
```suggestion
tcl112_ns = cg.esphome_ns.namespace('tcl112')
```

Use the full namespace name (unless it's a really long one)